spec/requests/cats_spec.rb

```diff
@@ -2,6 +2,92 @@
 
 RSpec.describe "Cats", type: :request do
   describe "GET /index" do
-    pending "add some examples (or delete) #{__FILE__}"
+    it "gets a list of cats" do
+      Cat.create(
+        name: 'Jax',
+        age: 5,
+        enjoys: 'Walks in the park',
+        image: 'https://images.unsplash.com/photo-1529778873920-4da4926a72c2?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1036&q=80'
+      )
+
+      get '/cats'
+      cat = JSON.parse(response.body)
+      expect(response).to have_http_status(200)
+      expect(cat.length).to eq(1)
+    end
+  end
+
+  describe "POST /create" do
+    it "creates a cat" do
+      cat_params = {
+        cat: {
+          name: 'Cherry',
+          age: 4,
+          enjoys: 'Eating and plenty of sunshine.',
+          image: 'https://images.unsplash.com/photo-1529778873920-4da4926a72c2?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1036&q=80'
+        }
+      }
+  
+    post '/cats', params: cat_params
+    expect(response).to have_http_status(200)
+    cat = Cat.first
+    expect(cat.name).to eq ('Cherry')
+    expect(cat.age).to eq (4)
+    expect(cat.enjoys).to eq ('Eating and plenty of sunshine.')
+    expect(cat.image).to eq ('https://images.unsplash.com/photo-1529778873920-4da4926a72c2?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1036&q=80')
+  end
+  end
+
+  describe "PATCH /update" do
+    it "can update an existing cat" do
+      cat_params = {
+        cat: {
+          name: 'Cherry',
+          age: 4,
+          enjoys: 'Eating and plenty of sunshine.',
+          image: 'https://images.unsplash.com/photo-1529778873920-4da4926a72c2?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1036&q=80'
+        }
+      }
+      post '/cats', params: cat_params
+      cat = Cat.first
+
+      update_cat_params = {
+        cat: {
+          name: 'Cherry',
+          age: 3,
+          enjoys: 'Eating and plenty of sunshine.',
+          image: 'https://images.unsplash.com/photo-1529778873920-4da4926a72c2?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1036&q=80'
+        }
+      }
+      patch "/cats/#{cat.id}", params: update_cat_params
+      updated_cat = Cat.find(cat.id)
+      expect(response).to have_http_status(200)
+      expect(updated_cat.age).to eq(3)
+    end 
+  end
+
+  describe "DELETE /destroy" do
+    it 'can delete an existing cat' do
+      cat_params = {
+        cat: {
+          name: 'Cherry',
+          age: 4,
+          enjoys: 'Eating and plenty of sunshine.',
+          image: 'https://images.unsplash.com/photo-1529778873920-4da4926a72c2?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1036&q=80'
+        }
+      }
+      post '/cats', params: cat_params
+      expect(response).to have_http_status(200)
```

Comment:
Since you are testing delete, you don't need the expect statements on line 80 and 82.  The post is simply so you have a cat created so you can destroy it.